modules/dpu-rhel-installer-node-with-podman.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/dpu-hardware-offload.adoc
+
+:_content-type: PROCEDURE
+[id="installing-rhel-installer-node-with-podman_{context}"]
+= Install Assisted Installer on the installer node using Podman
+
+Use Podman to install the assisted installer on your installer node. 
+
+.Prerequisites
+
+* You have Podman version 3.3+ installed
+
+.Procedure
+
+. Download link:https://github.com/openshift/assisted-service/blob/master/deploy/podman/pod.yml[pod.xml] and link:https://github.com/openshift/assisted-service/blob/master/deploy/podman/configmap.yml[configmap.yml] 
+
+. Update `IMAGE_SERVICE_BASE_URL` and `SERVICE_BASE_URL` in `configmap.yml` to match the hostname or IP address of your host.
```

Comment:
we should update more parameters to allow it to work (cause of arm support). Lets take defaults from 
https://github.com/eifrach/ansible-role-assisted-installer-pod
Maybe we can create our own pod.yml and configmap.yml ? 
@romfreiman what do you think?